pkg/domain/entities/play.go

```diff
@@ -64,6 +64,10 @@ type PlayKubeOptions struct {
 	Force bool
 	// PublishPorts - configure how to expose ports configured inside the K8S YAML file
 	PublishPorts []string
+	// Wait indicates whether to return after having created the pods
+	Wait bool
+	// PrintReport - prints the report from kube play if set to true
+	PrintReport bool
```

Comment:
That won't work for remote.  We need to keep printing in `cmd`.  It's a clearer separation of concerns.

Reply:
We need to print in the `PlayKube` function when --wait=true, otherwise the report is printed after the function has returned, so the user won't get any output of what containers and pods were created. 

I have added print to both the `PlayKube` function and in `cmd` based on whether `playOptions.PrintReport` is set to true or false. When --wait=true, PrintReport will always be true. This way we are able to give output to the user for the --wait case and don't break remote as well.